README.md

```diff
@@ -237,3 +237,17 @@ Methylation: a DNA modification where methyl groups of are added to cytosines in
 Molecular aberration type: encapsulates the different changes that occur the DNA, RNA and other biomolecules during cancer. For example: CNAs, SNVs, methylation, mRNA abundance, miRNA abundance.
 
 SNV: Single Nucleotide Variant. a nucleotide base (A,C,G,T) change in DNA.
+
+## License
+
+Author: Natalie Fox, Paul C. Boutros(pboutros@mednet.ucla.edu)
```

Comment:
**comment (non-blocking)**: I think we've also generally decided to not include email addresses in the READMEs unless there's a reason to include them for R packages?